build.sbt

```diff
@@ -263,6 +263,38 @@ lazy val roscala = (project in file("roscala"))
     libraryDependencies ++= commonDependencies
   ).dependsOn(roscala_macros)
 
+lazy val blockStorage = (project in file("block-storage"))
+  .enablePlugins(SiteScaladocPlugin, GhpagesPlugin, TutPlugin)
+  .settings(commonSettings: _*)
+  .settings(
+    name := "block-storage",
+    version := "0.0.1-SNAPSHOT",
+    libraryDependencies ++= commonDependencies ++ protobufLibDependencies ++ Seq(
+      lmdbjava,
+      catsCore,
+      catsEffect,
+      catsMtl
+    ),
+    /* Publishing Settings */
```

Comment:
You can remove this whole bit from here down to homepage, as we likely won't be publishing this to Sonatype.